src/wasm/wasm-debug.cpp

```diff
@@ -801,7 +815,7 @@ static void updateDIE(const llvm::DWARFDebugInfoEntry& DIE,
 
 static void updateCompileUnits(const BinaryenDWARFInfo& info,
                                llvm::DWARFYAML::Data& yaml,
-                               const LocationUpdater& locationUpdater) {
+                               LocationUpdater& locationUpdater) {
```

Comment:
why was this `const` removed? I think we should be able to keep it const here. Perhaps we need to add a const on updateDIE or elsewhere?

Reply:
I removed `const` there because I would like `updateDIE` to be able to modify locationUpdater, adding elements to debugLocToUnitMap.

Reply:
Oh, I see. Sounds good as it is then.

Reply:
Maybe worth adding a comment on top of the function to say that it updates the LocationUpdater?

Reply:
Done.